README.md

```diff
@@ -41,8 +41,10 @@ IPFS þing Jul 2022         |  IPFS Camp Oct 2022
 - [Background](#background)
 - [Installation](#installation)
   - [MacOS](#macos)
+    - [Go Client](#go-client)
   - [Linux](#linux)
```

Comment:
```suggestion
  - [Go Client](#go-client)
    - [MacOS](#macos)
    - [Linux](#linux)
```

Aren't both the installation instructions, MacOS and Linux, only for the Go Client?

Reply:
Good catch. Fixed with https://github.com/dennis-tra/punchr/pull/53/commits/1771625cd81093a20921333a5751247e60515f2e.